scene/2d/parallax_2d.cpp

```diff
@@ -123,13 +123,10 @@ void Parallax2D::_update_scroll() {
 }
 
 void Parallax2D::_update_repeat() {
-	if (!is_inside_tree()) {
-		return;
```

Comment:
Why did you remove this?

Reply:
Someone commented on another one of my PRs that early returns should only be done if the logic after is more than a couple lines or can branch in different ways. I think it was for `_validate_property()` and `_camera_moved()`. Since the change shortens it to only two lines now, I imagined someone would request it to be inside the if. It's not important to me, so whichever you prefer.

Reply:
Well it was already there, so it's fine to leave it.